audformat/core/database.py

```diff
@@ -314,16 +314,36 @@ def drop_tables(
             self,
             table_ids: typing.Union[str, typing.Sequence[str]],
     ):
-        r"""Drop tables by ID.
+        r"""Drop (miscellaneous) tables by ID.
 
         Args:
             table_ids: table IDs to drop
 
+        Raises:
+            audformat.errors.BadIdError: if a table with provided ID
+                does not exist in the database
+            RuntimeError: if a misc table with provided ID
+                is used as scheme(s)
+
         """
-        if isinstance(table_ids, str):
-            table_ids = [table_ids]
+        table_ids = audeer.to_list(table_ids)
         for table_id in table_ids:
-            self.tables.pop(table_id)
+            if table_id in self.tables:
+                self.tables.pop(table_id)
+            elif table_id in self.misc_tables:
+                schemes = self.misc_tables[table_id]._used_in_schemes()
```

Comment:
Or can't we simply do:

```python
                schemes = [
                    scheme._id for scheme in self.schemes.values()
                    if scheme.labels == table_id
                ]
```

Reply:
Very good idea, I removed `_used_in_schemes()` and use your solution now.